app/models/direct_file_data.rb

```diff
@@ -604,6 +605,10 @@ def interest_reported_amount=(value)
     write_df_xml_value(__method__, value)
   end
 
+  def qualifying_children_under_age_ssn_count=(value)
+    write_df_xml_value(__method__, value)
+  end
+
```

Comment:
how do you decide when to create a writer for a given field?

Reply:
in this case i made it so i could use it in a test 😬